examples/README.md

```diff
@@ -0,0 +1,3 @@
+# Terraform Equinix Kubernetes Addons - Blueprints
```

Comment:
We've been namespacing our examples. Is this intended to be an examples root README?

What content are you thinking would go here?

Reply:
All the examples we have added so far are specific to each addon. This examples folder is intended for opinionated blueprints/solutions that may use several addons as well as infrastructure and k8s clusters. I added that README file just to keep the empty `examples` folder but and I think it could include a short description like the one I have set here

Reply:
A `.keep` file would have worked 😏